### ISSUE TITLE
Menu entries not readable - add contrast

### ISSUE THREAD

**Issue by bugsysop** [BDX-town/Mangane#181]
![ui-menu-bug-social@tmprs](https://user-images.githubusercontent.com/365281/215206781-8af3e775-4436-4b6d-9623-e663b81fe484.png)

**Comment by Dehelssey** [comment]
Definitely somme huge accessibility issues. I'm working on it while redesigning some parts of the app ! Thanks for spotting that critical piece of UI !

**Comment by Dehelssey** [comment]
Also we have an issue with those cropped labels. They should rather use two lines if needed !

**Comment by avalanchemarramrat** [comment]
I am encountering the same issue.
![Screenshot from 2023-04-04 18-26-46](https://user-images.githubusercontent.com/129845076/229748891-50173dbb-7a53-46b7-acb8-6eac3ba42cef.png)

**Comment by Dehelssey** [comment]
This issue is about legibility / low contrast on menu.

The issue about truncated text is #184

**Comment by Cl0v1s** [comment]
Idem for white theme
<img width="279" alt="image" src="https://github.com/user-attachments/assets/29fb4573-5682-4e00-a793-3280519a9a41">
